### PR TITLE
Add stake or unstake dialog

### PR DESCRIPTION
**Pull request by codestar73**
- Stake Dialog
Screen Shot 2022-06-22 at 12 11 48 AM
image

### CODE REVIEW

File: src/pages/Farm/StakeDialog.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  onClose: () => void
+  farmName: string
+}
+
+export default function StakeDialog({
+  open,
+  farmName,
+  onClose,
+}: StakeDialogProps): JSX.Element {
+  const [stakeStatus, setStakeStatus] = useState<"stake" | "unStake">("stake")
```
Comment (ug02fast) [inline src/pages/Farm/StakeDialog.tsx:25]: unstake is one word, I think better to have it `unstake`

File: src/pages/Farm/StakeDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+              Claim
+            </Button>
+          </Stack>
+          <Tabs
+            value={stakeStatus}
+            variant="fullWidth"
+            onChange={(_, newValue) => setStakeStatus(newValue)}
+          >
+            <Tab value="stake" label="Stake" />
+            <Tab value="unStake" label="Unstake" />
```
Comment (ug02fast) [inline src/pages/Farm/StakeDialog.tsx:59]: `value=unstake`?

Comment (nkrulikovsky) [reply src/pages/Farm/StakeDialog.tsx:59]: This is how `Tabs` works in MUI. It is a little tricky, but when click `Tab` the newValue on the function of `onChange` in `Tabs` will transfer this value

Comment (ug02fast) [reply src/pages/Farm/StakeDialog.tsx]: that's fine, I just wanted to make sure you saw the casing on `unStake -> unstake`

**Review by hammeiam — APPROVED** [review]
- The pair's name should be SDL/WETH
- I know the TokenInput wasn't built to handle this, but if possible, I think the token passed to TokenInput should have a symbol `SLP` (Sushi Liquidity Pair), and the name SDL/WETH SLP. It should probably also use the sushi icon https://assets.coingecko.com/coins/images/12271/small/512x512_Logo_no_chop.png?1606986688

**Comment by nkrulikovsky** [comment]
@hammeiam 
Updated TokenInput!

**Review by ug02fast — APPROVED** [review]

**Pull request merged**